apps/ledger-live-desktop/src/renderer/drawers/DataSelector/AccountList.tsx

```diff
@@ -78,6 +85,8 @@ export function AccountList({ currency, onAccountSelect, accounts$ }: Props) {
       {accountTuples.map((accountTuple, index) => {
         const { account, subAccount } = accountTuple;
         const accountCurrency = getAccountCurrency(subAccount || account);
+        const unit = currencySettings.unit;
```

Comment:
code is no longer equivalent.
unit is supposed to be inferred from the `accountCurrency` but here you use a global currencySettings one that is fixed for one currency? do we have guarantee it's the same currency?